ultra/ultra/evaluate.py

```diff
@@ -53,21 +53,22 @@ def evaluation_check(
     timestep_sec,
     headless,
     log_dir,
+    evaluation_task_ids,
 ):
     # Evaluate agents that have reached the eval_rate.
     agent_ids_to_evaluate = [
         agent_id
         for agent_id in agent_ids
-        if (episode.get_itr(agent_id) + 1) % eval_rate == 0
-        and episode.last_eval_iterations[agent_id] != episode.get_itr(agent_id)
+        if episode.index % eval_rate == 0
```

Comment:
Just curious, should this be:
```suggestion
        if (episode.index + 1) % eval_rate == 0
```
Otherwise, the way it is currently (without the "`+ 1`"), I think it will always evaluate on the first episode (when `episode.index == 0`). Is this what we want?

Reply:
I think it's good practiceto evaluate at initialization, it provides a baseline before any learning happens.